bspacks/reason_bspack.sh

```diff
@@ -7,17 +7,17 @@ set -e
 
 THIS_SCRIPT_DIR="$(cd "$( dirname "$0" )" && pwd)"
 
-echo "**This script is switching you to ocaml 4.02.3 for the subsequent bspacking. Please switch back to your own version afterward. Thanks!**\n"
-opam switch 4.02.3
-eval $(opam config env)
+cd $THIS_SCRIPT_DIR
+
+# Install & build menhir, omp
+esy i
```

Comment:
Isn't
```
esy install
esy build
```
the same as just `esy` now? But I maybe we want it more explicit?

Reply:
Yes. However, we might find some issues with the `esy=esy i && esy b` shortcut so it's good to be explicit in case it changes.